src/main/frontend/handler/events.cljs

```diff
@@ -82,7 +82,8 @@
     (let [result (async/<! (sync/<user-info sync/remoteapi))]
       (cond
         (instance? ExceptionInfo result)
-        nil
+        (do (notification/show! ":user/fetch-info-and-graphs failed" :warning)
```

Comment:
The ExceptionInfo could be either an error from our server or an unstable network, right?
If so, we should handle both cases. For server errors, we can say `Something is wrong, please try again.` and use Sentry to record the error. If it's from an unstable network, we can probably retry it and finally notice the user about unstable network if it still doesn't work.